netlify.toml

```diff
@@ -1070,4 +1081,14 @@ status = 301
 [[redirects]]
 from = "/workflows/why-to-use-workflows"
 to = "/workflows"
+status = 301
+
+[[redirects]]
+from = "/integrations/appdome-integration"
+to = "/workflows/ios-specific-workflow-steps/appdome-build-to-secure-for-ios"
+status = 301
+
+[[redirects]]
+from = "/integrations/jira-integration"
+to = "/workflows/common-workflow-steps/jira-comment"
```

Comment:
I checked this reference in docs and it's currently a circular redirect.

![Screenshot 2024-12-02 at 14 30 24](https://github.com/user-attachments/assets/7b5958bd-48f0-461b-87ba-d2438ec54ef9)

Here we're redirecting the document itself and it's not helping user.

The URL here should be below which explains the variable itself.
- /workflows/common-workflow-steps/jira-comment#configuration-of-jira-comment

We do not need Netlify here, we should change the `jira-comment.md` only.